lib/user32-sys/src/lib.rs

```diff
@@ -94,7 +94,7 @@ extern "system" {
     pub fn CharUpperBuffA(lpsz: LPSTR, cchLength: DWORD) -> DWORD;
     pub fn CharUpperBuffW(lpsz: LPWSTR, cchLength: DWORD) -> DWORD;
     pub fn CharUpperW(lpsz: LPWSTR) -> LPWSTR;
-    // pub fn CheckDlgButton();
+    pub fn CheckDlgButton(hDlg: HWND,nIDButton: c_int,uCheck: UINT) -> BOOL;
```

Comment:
The script below use "regexp replace" to get all things done. I hope more contributors can use it to format code quickly and avoid spent too much time on this problem.And please help improve it if someone finds a bug.

(1) Spaces after commas
",([^ \r\n])"   replace by   ", \1"
(2) space between colon and type
"([^:]):([^: ])"  replace by   "\1: \2"
(3) spaces around -> 
"([^ ])->"   replace by    "\1 ->"
"->([^ ])"   replace by    "-> \1"

This was tested on notepad++ 6.7.5